paasta_tools/marathon_tools.py

```diff
@@ -277,6 +277,7 @@ class FormattedMarathonAppDict(BounceMethodConfigDict, total=False):
     env: Dict[str, str]
     mem: float
     cpus: float
+    gpus: float
```

Comment:
can you really have a float? does that even work?

Reply:
Yeah, it works. Look for dsc-test-service.gpu in pnw-devc. I am using float here hoping gpus can be shared in the near future, i.e. a task can have 0.5 gpus.